### ISSUE TITLE
SwiftUI Host View is too big (segmented control has lots of white space below it)

### ISSUE THREAD

**Issue by Mozahler** [yonat/MultiSelectSegmentedControl#40]
I want to put a bar at the top of the screen, like the demo. The non-swiftui version works perfectly, but the SwiftUI version has a lot of space below the actual bar:
<img width="293" alt="Screen Shot 2020-04-16 at 11 21 23 AM" src="https://user-images.githubusercontent.com/9641531/79474825-e5989d80-7fd4-11ea-98eb-dfac1bfddcd0.png">

How do I shrink the blue area down to the height of the control?

**Minimal project that reproduces the problem (so I'll be able to figure out how to fix it):**
I have opened a question on StackOverflow with more details.

[StackOverflow](https://stackoverflow.com/questions/61254931/how-do-i-get-my-uiviewrepresentable-to-correctly-size-for-its-content)

**Comment by Mozahler** [comment]
I didn't mean to call it a bug. How do I remove that bug label?

**Comment by Mozahler** [comment]
Nevermind, I can close it.
Here's how to collapse it (see "add this here"):
  ```
  var body: some View {
        VStack(alignment: .center) {
            Spacer()
            MultiSegmentPicker(
                selectedSegmentIndexes: $selectedSegmentIndexes,
                items: ["First", "Second", "Third", "Done"]
            )
            .fixedSize() <=== Add this here
        }
    }
```

**Comment by Mozahler** [comment]
this worked in the demo, but in a real app, with views below it, it disappears behind the other views. the view debugger says: runtime: Layout Issues: Position is ambiguous for MultiSelectSegmentedControl. I can see the controller where it belongs, but the next view in the VStack sits on top of it.

It's hard to see, but there is a bar with blue "A" "b" "C" in the center near the top. It is hidden behind "Unreviewed" and "Reviewed"

<img width="285" alt="abc" src="https://user-images.githubusercontent.com/9641531/79488952-9197b400-7fe8-11ea-9d54-8e25c3d4cd5d.png">
<img width="780" alt="hidden abc" src="https://user-images.githubusercontent.com/9641531/79488956-92c8e100-7fe8-11ea-87df-e8c474b755e9.png">

**Comment by Mozahler** [comment]
I wrote a second question up in StackOverflow:
https://stackoverflow.com/questions/61257744/uiviewrepresentable-hidden-behind-swiftui-view

I think the problem is with how constraints are being set in the **MultiSelectSegmentedControl** class.

**Comment by yonat** [comment]
Fixed in 2.3.3 - `fixedSize()` should work fine now.
